demos/tests/run_tests.py

```diff
@@ -65,6 +65,8 @@ def parse_args():
         help='path to report file')
     parser.add_argument('--suppressed-devices', type=Path, required=False,
         help='path to file with suppressed devices for each model')
+    parser.add_argument('--precisions', type=str, nargs='+', default=['FP16', 'FP32'],
+        help='IR precisions for all models. By default, models are tested in FP32 precision')
```

Comment:
```suggestion
    parser.add_argument('--precisions', type=str, nargs='+', default=['FP16'],
        help='IR precisions for all models. By default, models are tested in FP16 precision')
```

better to have FP15 as default, it will take less time. Also, not sure, if we set default as ['FP16','FP16-INT8','FP16-INT1'], will this enforce to run all models which have INT8 variant too?